ironfish/src/fileStores/config.ts

```diff
@@ -279,6 +279,66 @@ export type ConfigOptions = {
    * Always allow incoming connections from these IPs even if the node is at maxPeers
    */
   incomingWebSocketWhitelist: string[]
+
+  /**
+   * Enable the node's in-process wallet to scan blocks and mempool transactions
+   */
+  enableWalletScanning: boolean
```

Comment:
nit: I like how the other flags are prefixed with `wallet` followed by the config, so maybe something like `walletScanningEnabled` would be more consistent.